src/Mvc/Mvc.ApiExplorer/test/DefaultApiDescriptionProviderTest.cs

```diff
@@ -1579,6 +1583,120 @@ public void GetApiDescription_ParameterDescription_FromQueryEmployee()
         Assert.Equal(typeof(string), id.Type);
     }
 
+    [Fact]
+    public void GetApiDescription_ParameterDescription_ParsablePrimitiveType()
```

Comment:
Do we have versions of these tests in `EndpointApiDescriptionProvider`?

Reply:
Those tests give us a similar coverage:

https://github.com/dotnet/aspnetcore/blob/c8d252dc43180b1718caae385bda65fed13e7822/src/Mvc/Mvc.ApiExplorer/test/EndpointMetadataApiDescriptionProviderTest.cs#L273-L327

Reply:
It'd be nice to have regression tests for the types we're special-casing like `Guid`.

Reply:
Added `AddsFromRouteParameterAsPathWithSpecialPrimitiveType` and `AddsFromRouteParameterAsPathWithNullableSpecialPrimitiveType`